database/models/search.js

```diff
@@ -0,0 +1,36 @@
+const client = require('../index');
+
+const search = {
+  get: (req, res) => {
+    const { query } = req.query;
+    client.query(`
+      SELECT json_build_object(
+        'artists', (
+          SELECT array_to_json(array_agg(json_build_object( 
+          'id', a.id, 
+          'name', a.display_name, 
+          'pic', a.pic )))
+          FROM artists a
+          WHERE a.display_name LIKE '%${query}%'
+          OR a.instrument LIKE '%${query}%'
+          OR a.genre LIKE '%${query}%'
```

Comment:
We should add bio in this, I think. But I will approve for now because I want to test against frontend.